src/main/java/com/google/devtools/build/lib/analysis/config/CoreOptionConverters.java

```diff
@@ -234,6 +234,42 @@ public String getTypeDescription() {
     }
   }
 
+  /**
+   * Flag converter for canonicalizing a label (possibly with a "/..." suffix) and/or define by
+   * converting the label to ambiguous canonical form.
+   */
+  public static class CustomFlagConverter implements Converter<String> {
+    @Override
+    public String convert(String input, Object conversionContext) throws OptionsParsingException {
+      if (!input.startsWith("//") && !input.startsWith("@")) {
+        // This is a --define flag.
+        return input;
+      }
+      // A "/..." suffix is not valid label syntax, so replace it with valid syntax and transform
+      // it back after conversion.
+      String invalidSubpackagesSuffix = "/...";
```

Comment:
Make these constants in the `CustomFlagConverter` class.

Reply:
I made `/...` a constant as it has meaning, but `:__subpackages__` is arbitrary and not used anywhere else, so defining a constant for it may, in my opinion, even hurt readability.